packs/loaders/testimonials.ts

```diff
@@ -0,0 +1,61 @@
+import { Context } from "$store/packs/accounts/configStore.ts";
+import paths from "$store/packs/utils/paths.ts";
+import {
+  APIGetTestimonials,
+  ProductData,
+  Review,
+  TestimonialProps,
+} from "deco-sites/otica-isabela/packs/types.ts";
+import { fetchAPI } from "deco-sites/std/utils/fetch.ts";
+import { toReview } from "deco-sites/otica-isabela/packs/utils/transform.ts";
+
+/**
+@title Otica Isabela Dias Testimonials getter
```

Comment:
Segue esse padrão nos nomes dos Loaders por favor.
```suggestion
@title Otica Isabela Dias - Testimonials
```